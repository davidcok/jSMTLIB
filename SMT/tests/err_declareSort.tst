; declare-sort with malformed arguments
(set-logic QF_UF)
(declare-sort)
(declare-sort A)
(declare-sort A x)
(declare-sort 1)
(declare-sort A 1 2)
