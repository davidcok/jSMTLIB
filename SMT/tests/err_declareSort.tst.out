success
(declare-sort)
             ^
(error "Expected a symbol here, not a )")
(declare-sort A)
               ^
(error "Expected a numeral here, instead of a )")
(declare-sort A x)
                ^
(error "Expected a numeral here, instead of a symbol")
(declare-sort 1)
              ^
(error "Expected a symbol here, not a numeral")
(declare-sort A 1 2)
                  ^
(error "Too many arguments or extraneous material after the command or missing right parenthesis")
